add-on/src/contentScripts/ipfs-proxy/page.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+const Ipfs = require('ipfs')
+const { createProxyClient } = require('ipfs-postmsg-proxy')
+const _Buffer = Buffer
+
+window.Buffer = window.Buffer || _Buffer
+window.Ipfs = window.Ipfs || Ipfs
```

Comment:
Maybe you want an IPFS node configured with particular swarm peers so using the one provided by the extension is not possible? ...and you don't want to download the IPFS lib because you're on a poor connection?

Maybe...for experiments?

Are they good enough?

Reply:
I'd say yes, it would make developer's life easier.  👍 
But perhaps it should be renamed  to `window.ipfsApiClient` or something like that?

@diasdavid, would appreciate your thoughts on this from `js-ipfs-api` perspective